utils/meta_infos.json

```diff
@@ -298,7 +339,11 @@
     "lener_br": "lener_br",
     "mapa_coarse": "mapa",
     "mapa_fine": "mapa",
-    "turkish_constitutional_court_decisions_judgment": "turkish_constitutional_court_decisions_judgment"
+    "turkish_constitutional_court_decisions_judgment": "turkish_constitutional_court_decisions_judgment",
+    "swiss_criticality_prediction_bge_facts": "swiss_criticality_prediction_bge_facts",
+    "swiss_criticality_prediction_bge_considerations": "swiss_criticality_prediction_bge_considerations",
+    "swiss_criticality_prediction_citation_facts": "swiss_criticality_prediction_citation_facts",
+    "swiss_criticality_prediction_citation_considerations": "swiss_criticality_prediction_citation_considerations"
```

Comment:
here they should all map to swiss_criticality_prediction

Reply:
I thought this is where I define the dataset config I want to use. If all map to swiss_criticality_prediction how can i use a different dataset config?

Reply:
I got it wrong and corrected it. We have now Swiss_Criticality_Prediction with 4 different tasks.